test/Splunk.OpenTelemetry.AutoInstrumentation.IntegrationTests/Helpers/EnvironmentHelper.cs

```diff
@@ -30,8 +30,6 @@
 // limitations under the License.
 // </copyright>
 
-#nullable disable
```

Comment:
minimal copy from upstream, I will cleanup other files in this folder in follow up pr